src/main/groovy/com/todo/dao/TaskDaoI.groovy

```diff
@@ -0,0 +1,12 @@
+package com.todo.dao
+
+import com.todo.db.sql.tables.pojos.Task
+
+import java.sql.Connection
+
+interface TaskDaoI {
```

Comment:
Appending I to interface names is usually not a widely adopted practice in Java unlike .Net.
However, some sections of people do it too. One of the reasons for not adopting it being reducing the readability of the name.